lib/dicom_net/association.ex

```diff
@@ -160,44 +183,75 @@ defmodule DicomNet.Association do
     %{name: _ts_name, options: ts_options} = Dicom.UidRegistry.get_transfer_syntax(ts_uid)
 
     # TODO this is not always c-store
+    ds = Dicom.BinaryFormat.from_binary(data, ts_options)
+    command_Code = DataSet.value_for!(command, :CommandField)
+
+    response =
+      case command_Code do
+        0x01 ->
+          Logger.debug("Start handling C-Store")
+
+          case Keyword.fetch(state.handlers, :cstore) do
+            :error ->
+              Logger.debug("No C-Store function defined. Returning Failed: Unable to process.")
+              # Returns Failed Unable to Process when no C-Store handler function is defined.
+              asci_de = DataSet.fetch!(command, :AffectedSOPClassUID)
+              mid_de = DataSet.fetch!(command, :MessageID)
+              # TODO: replace cfind_response_tail with a proper C-Store response.
+              response_ds = cfind_response_tail(asci_de, mid_de, @unable_to_process)
+
+              Dicom.BinaryFormat.serialize_command_data_set(response_ds)
+              |> Pdu.new_data_pdu(presentation_context_id, type: :command_last_fragment)
+              |> Pdu.serialize()
+
+            {:ok, cstore_Handler} ->
```

Comment:
I'm being nitpicky here now - but please just lower_snake_case for variable names and atoms: `cstore_handler`. Modules are UpperCamelCase. Keeping the code style consistent is important :)